File: app/src/main/kotlin/com/popalay/cardme/utils/animation/EndAnimatorListener.java
package com.popalay.cardme.utils.animation;

import android.animation.Animator;

public abstract class EndAnimatorListener implements Animator.AnimatorListener {

    @Override public void onAnimationStart(Animator animation) {}

    @Override public void onAnimationCancel(Animator animation) {}

    @Override public void onAnimationRepeat(Animator animation) {}
}

File: presentation/src/main/kotlin/com/popalay/cardme/utils/animation/EndAnimatorListener.kt
package com.popalay.cardme.utils.animation

import android.animation.Animator

interface EndAnimatorListener : Animator.AnimatorListener {

    override fun onAnimationStart(animation: Animator) = Unit

    override fun onAnimationCancel(animation: Animator) = Unit

    override fun onAnimationRepeat(animation: Animator) = Unit
}
